server/trillian_log_server/main.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/google/trillian/crypto/keys"
 	"github.com/google/trillian/extension"
 	"github.com/google/trillian/monitoring"
-	"github.com/google/trillian/quota"
+	mysqlq "github.com/google/trillian/quota/mysql"
```

Comment:
OK. I'm happy to do any changes required for multi-dialect support, just let me know what/when you need it.